PeakLocalisations/Localisation/App Store/Metadata.xcstrings

```diff
@@ -16,6 +22,12 @@
       "comment" : "Make sure to use the same exact link for the Terms of Use at the bottom",
       "extractionState" : "manual",
       "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Mit Peak kannst du deine eigene Fitness-Übersicht erstellen und mit einer Reihe von Widgets immer den Überblick behalten.\n\n→ Deine persönliche Fitness-Übersicht\nErstelle eine individuelle Übersicht, die auf deine Bedürfnisse zugeschnitten ist, indem du genau die Kennzahlen auswählst, die du möchtest, z. B. Schritte, Trainings, Aktivitätsringe und mehr. Jede Kennzahl kann eine Vielzahl von Blöcken enthalten, z. B. Diagramme, Ziele, Summen, Trends und mehr.\n\n→ Widgets für den Home- und Sperrbildschrim\nPeak enthält eine Vielzahl wunderschön gestalteter Widgets, mit denen du deine Fortschritte auf einen Blick sehen kannst. So kannst du deine Statistiken für den heutigen Tag, die Fortschritte deiner Ziele und vieles mehr im Auge behalten. Jedes Widget kann mit einem eigenen Theme für den hellen und dunklen Modus angepasst werden.\n\n→ Eine Auswahl an Themes\nPeak enthält über 60 wunderschön gestaltete Themes, damit du genau das Aussehen findest, das deinem Geschmack entspricht. Du kannst sogar verschiedene Themes für den hellen und den dunklen Modus einstellen.\n\n→ Privatsphäre von Anfang an\nDatenschutz ist einer der Grundwerte von Peak. Gesundheitsdaten gehören zu den sensibelsten Daten, die Menschen haben, deshalb gibt es kein Tracking und keine Datenerfassung.\n\n→ Angenehme Bedienung\nPeak fühlt sich auf deinem iPhone und iPad wie zu Hause und enthält vertraute Muster und viele kleine Details, die die Nutzung der App zu einem Kinderspiel machen. Du kannst verschiedene Aspekte anpassen, z. B. das App-Symbol, die Themes, die gewählten Einheiten und vieles mehr.\n\nNutzungsbedingungen: https://www.apple.com/legal/internet-services/itunes/dev/stdeula/"
```

Comment:
I think the plural is correct here, since it is about multiple workouts, not just one. And even though German imported that word, the plural is still "Trainings".

As for the added `an`, it would change the meaning of the sentence. It's about Peak feeling "right at home" (original) on your iPhone, since it uses the same patterns. It's more in the sense of 'it fits right in'.